ruleset/rules/0750-github_rules.xml

```diff
@@ -0,0 +1,3225 @@
+
+    <!-- billing.change_billing_email -->
+    <rule id="91113" level="9">
+        <if_sid>91111</if_sid>
+        <action>billing.change_billing_email</action>
+        <description>GitHub Billing change billing email.</description>
+        <options>no_full_log</options>
+        <group>git_billing</group>
+    </rule>
+
+    <!-- dependabot_alerts
+            disable
+            enable
+    -->
+    <rule id="91114" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^dependabot_alerts.\.*</action>
+        <description>GitHub Dependabot alerts.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_alerts</group>
+    </rule>
+
+    <!-- dependabot_alerts.disable -->
+    <rule id="91115" level="9">
+        <if_sid>91114</if_sid>
+        <action>dependabot_alerts.disable</action>
+        <description>GitHub Dependabot alerts disable.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_alerts</group>
+    </rule>
+
+    <!-- dependabot_alerts.enable -->
+    <rule id="91116" level="5">
+        <if_sid>91114</if_sid>
+        <action>dependabot_alerts.enable</action>
+        <description>GitHub Dependabot alerts enable.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_alerts</group>
+    </rule>
+
+    <!-- dependabot_alerts_new_repos
+            disable
+            enable
+     -->
+    <rule id="91117" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^dependabot_alerts_new_repos.\.*</action>
+        <description>GitHub Dependabot alerts new repos.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_alerts_new_repos</group>
+    </rule>
+
+    <!-- dependabot_alerts_new_repos.disable -->
+    <rule id="91118" level="9">
+        <if_sid>91117</if_sid>
+        <action>dependabot_alerts_new_repos.disable</action>
+        <description>GitHub Dependabot alerts new repos disable.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_alerts_new_repos</group>
+    </rule>
+
+    <!-- dependabot_alerts_new_repos.enable -->
+    <rule id="91119" level="5">
+        <if_sid>91117</if_sid>
+        <action>dependabot_alerts_new_repos.enable</action>
+        <description>GitHub Dependabot alerts new repos enable.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_alerts_new_repos</group>
+    </rule>
+
+    <!-- dependabot_security_updates
+            disable
+            enable
+    -->
+    <rule id="91120" level="3">
+        <if_sid>91100</if_sid>
+        <action type="osregex">^dependabot_security_updates.\.*</action>
+        <description>GitHub Dependabot security updates.</description>
+        <options>no_full_log</options>
+        <group>git_dependabot_security_updates</group>
+    </rule>
+
+    <!-- dependabot_security_updates.disable -->
+    <rule id="91121" level="9">
```

Comment:
level 9 is "Error from invalid source", perhaps level 12 "High importance event" is more appropiate